app/stories/pix-toast.stories.js

```diff
@@ -0,0 +1,75 @@
+import { hbs } from 'ember-cli-htmlbars';
+
+export default {
+  title: 'Notification/Toast',
+  render: (args) => ({
+    template: hbs`<PixToast
+  @type={{this.type}}
+  @showToast={{this.showToast}}
+  @ariaLabelForCloseButton='Fermer'
+  @onCloseButtonClick={{fn (mut this.showToast) (not this.showToast)}}
```

Comment:
Comme le bouton de fermeture n'est affiché qu'avec la notification, la seule option possible est de fermer la notification.
Je propose le code suivant :
```suggestion
  @onCloseButtonClick={{fn (this.showToast false)}}
```

Autre suggestion, ne pas rendre obligatoire l'argument `onCloseButtonClick` et fermer la notification par défaut lors d'un clic sur le bouton `close`.

Reply:
On est entrain de tout reprendre donc ce paramètre ne va probablement pas rester